pom.xml

```diff
@@ -99,10 +100,10 @@
 	</modules>
 
 	<properties>
-		<camel-spring-boot-version>4.10.0.redhat-00001</camel-spring-boot-version>
+		<camel-spring-boot-version>4.10.0.redhat-00002</camel-spring-boot-version>
```

Comment:
Can you remove the changes to the pom.xml and rebase on the commit just merged with more recent jkube / camel-spring-boot versions?   https://github.com/jboss-fuse/camel-spring-boot-examples/pull/172

Reply:
done thanks